src/main/scala/Exercise.scala

```diff
@@ -0,0 +1,22 @@
+class Exercise {
+  def FirstSolution(){
```

Comment:
method names start with a lowercase word, only words after should have capitalised first letter